runtime/themes/adwaita-light.toml

```diff
@@ -0,0 +1,110 @@
+inherits="adwaita-dark"
+"attribute" = "orange_6"
```

Comment:
It looks like this isn't defined in `adwaita-dark`'s palette or the palette below

Reply:
that's a bit embarrassing haha!

it's fixed now - tried it on a html file and it looks better/more colorfull